backend/src/main/java/com/bakdata/conquery/models/preproc/Preprocessed.java

```diff
@@ -70,6 +62,8 @@ public Preprocessed(ConqueryConfig config, PreprocessingJob preprocessingJob) th
 		this.descriptor = preprocessingJob.getDescriptor();
 		this.name = this.descriptor.getName();
 
+
+		//TODO this may cause an error if the structure of the first input is different to the others, so maybe it is better to consider other inputs too
```

Comment:
Die anderen Inputs könenn eigentlich nicht anders sein, das wäre eine mis-konfiguration und wird in der Validierung abgefangen: `com.bakdata.conquery.models.preproc.TableImportDescriptor#isSameTypesInEachInput`